server/routes/prisonerList/prisonerListView.ts

```diff
@@ -63,6 +63,11 @@ const buildItemsArray = (config: {
   sortOrder: string
 }): Item[] => {
   const items: Item[] = []
+
```

Comment:
I made this change to the View class so that if there is only 1 record in the items array (and therefore it must be page 1 / current page), then return an empty array instead.
This was to prevent a lone `1` being displayed as the pagination controls, and instead there are no pagination controls and just the record count (see screenshot in main description)